owncloudApp/src/main/res/layout/spaces_list_item.xml

```diff
@@ -53,16 +58,37 @@
                 android:layout_height="1dp"
                 android:background="@color/actionbar_start_color" />
 
-            <TextView
-                android:id="@+id/spaces_list_item_name"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:layout_margin="@dimen/standard_half_margin"
-                android:singleLine="true"
-                android:textColor="@color/textColor"
-                android:textSize="18sp"
-                android:textStyle="bold"
-                tools:text="Name" />
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:orientation="horizontal">
+
+                <TextView
+                    android:id="@+id/spaces_list_item_name"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="1"
+                    android:layout_margin="@dimen/standard_half_margin"
+                    android:singleLine="true"
+                    android:textColor="@color/textColor"
+                    android:textSize="18sp"
+                    android:textStyle="bold"
+                    tools:text="Name" />
```

Comment:
I miss some content description for accessibility. We should not forget TalkBack ;)

Reply:
💯